lwm2m-client/src/main.c

```diff
@@ -0,0 +1,452 @@
+#include "sol-log.h"
```

Comment:
these includes of soletta should be using `<` and `>`. I forgot to check on the other PRs, but we should have this solved at the end (maybe different PR).